cgroups/fs/cpuset.go

```diff
@@ -48,6 +48,11 @@ func (s *CpusetGroup) GetStats(path string, stats *cgroups.Stats) error {
 }
 
 func (s *CpusetGroup) ApplyDir(dir string, cgroup *configs.Cgroup, pid int) error {
```

Comment:
Shouldn't we not even make the call to apply for this subsystem if it doesn't exist?